### PR TITLE
Standardized Readme

### PR DESCRIPTION
**Pull request by RichardLitt**
See multiformats/multiformats#13

### CODE REVIEW

**Comment by daviddias** [comment]
Thank you @RichardLitt , just needing a rebase :)

File: README.md [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
-==================
+# multistream-select
 
 [![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
-[![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
+[![](https://img.shields.io/badge/project-multiformats-blue.svg?style=flat-square)](http://github.com/multiformats/multiformats)
 [![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
 
-> Friendly protocol multiplexing. It enables a multicodec to be negotiated between two entities.
+> self-describing protocol streams
```
Comment (RichardLitt) [inline README.md]: Updated this to match the repo. Should we change the repo description back?

File: README.md [hunk 2]
```diff
@@ -163,8 +176,21 @@ For example
...
+## Contribute
+
+Contributions welcome. Please check out [the issues](https://github.com/multiformats/multistream-select/issues).
+
+Check out our [contributing document](https://github.com/multiformats/multiformats/blob/master/contributing.md) for more information on how we work, and about contributing in general. Please be aware that all interactions related to multiformats are subject to the IPFS [Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
+
+## License
 
-- [js-multistream-select](https://github.com/diasdavid/js-multistream-select) - JavaScript Implementation
-- [go-multistream-select](https://github.com/whyrusleeping/go-multistream-select) - Go Implementation
```
Comment (RichardLitt) [inline README.md]: This go link is dead. Is there a link, elsewhere? Couldn't find it.

File: README.md [hunk 3]
```diff
@@ -163,8 +176,21 @@ For example
...
 
-# Implementations
+## Implementations
+
+- [js-multistream-select](https://github.com/multiformats/js-multistream-select) - JavaScript Implementation
+- [mss-nc](https://github.com/whyrusleeping/mss-nc) - multistream netcat written in Go
+
+## Maintainers
+
+Captain: [@jbenet](https://github.com/jbenet).
```
Comment (RichardLitt) [inline README.md]: Are you the captain, @diasdavid?

**Comment by RichardLitt** [comment]
Rebased, added some comments. Squash before merge, somehow the history is a bit muddy.

File: README.md (hunk 1 above)
Comment (daviddias) [reply README.md]: Just updated the repo description to match the new one. 

```
Friendly protocol multiplexing. It enables a multicodec to be negotiated between two entities.
```

File: README.md (hunk 3 above)
Comment (daviddias) [reply README.md]: At this point, seems so, yes :)

File: README.md (hunk 2 above)
Comment (daviddias) [reply README.md]: Oh right, it is just go-multistream
https://github.com/multiformats/go-multistream

**Comment by daviddias** [comment]
thank you @RichardLitt :)

**Comment by RichardLitt** [comment]
@diasdavid I think this is ready now?

**Pull request merged**